decidim-core/lib/decidim/page_finder.rb

```diff
@@ -0,0 +1,48 @@
+# frozen_string_literal: true
+module Decidim
+  # A custom implementation for HighVoltage so we can correctly find which page
+  # to render. We need this because we allow rendering pages with content from
+  # the database (with Decidim::Page) but also fallback to a template if it
+  # exists.
+  class PageFinder < HighVoltage::PageFinder
```

Comment:
Are you entirely sure using `HighVoltage` is justified? In the end, it only ends up doing something like `render template: "pages/#{page_name}"` in the controller. 

We're including a dependency that doesn't add much value and that can lead to issues in the future (being locked to a Rails version, for example).